tests/test_feature_extraction_speech_to_text.py

```diff
@@ -137,15 +137,9 @@ def test_cepstral_mean_and_variance_normalization(self):
         feature_extractor = self.feature_extraction_class(**self.feat_extract_tester.prepare_feat_extract_dict())
         speech_inputs = [floats_list((1, x))[0] for x in range(800, 1400, 200)]
 
-        # TODO(Patrick, Suraj, Anton) - It's surprising that "non-padded/non-numpified" padding
-        # results in quite inaccurate variance computation after (see 5e-1 tolerance)
-        # Issue is filed and PR is underway: https://github.com/huggingface/transformers/issues/13539
-        #        paddings = ["longest", "max_length", "do_not_pad"]
-        #        max_lengths = [None, 16, None]
-        #        var_tolerances = [1e-3, 1e-3, 5e-1]
-        paddings = ["longest", "max_length"]
-        max_lengths = [None, 16]
-        var_tolerances = [1e-3, 1e-3]
+        paddings = ["longest", "max_length", "do_not_pad"]
+        max_lengths = [None, 16, None]
+        var_tolerances = [1e-3, 1e-3, 1e-1]
```

Comment:
still need `1e-1` tolerance?

Reply:
`1e-3` doesn't work?

Reply:
Actually it does work, thanks for catching this! :)